### PR TITLE
Inline react app rewired instructions

### PR DESCRIPTION
**Pull request by ramonh**
In order to avoid any issues with tests, This inlines the instructions from React App Rewired, adapted for CodeSee

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -54,16 +54,25 @@ We have specific instructions for certain projects and environments. If you are
 
 ### Configuring CodeSee with Create React App
 
-**Add React App Rewired**
+Add [React App Rewired](https://github.com/timarney/react-app-rewired#how-to-rewire-your-create-react-app-project) to your project as described below:
```
Comment (jleven) [inline docs/index.md:58]: I think it's confusing to both link to their instructions, and copy them below. Won't some people click the link and follow that?

Can we be more explicit about that (note I changed the link so it no longer points at the install instructions):

> We're going to add [React App Rewired](https://github.com/timarney/react-app-rewired) following the instructions below. Please note that these instructions are slightly different than the ones on the react-app-rewired site. But, if you already have react-app-rewired installed, nothing to worry about here, you can skip to the next step.

File: docs/index.md [hunk 2]
```diff
@@ -101,6 +110,32 @@ module.exports = function override(config, env) {
...
+|   +-- config-overrides.js
+|   +-- node_modules
+|   +-- package.json
+|   +-- public
+|   +-- README.md
+|   +-- src
+```
+
+#### 3) 'Flip' the existing calls to `react-scripts` in `npm` scripts for start, build and test
+```diff
```
Comment (jleven) [inline docs/index.md:125]: This says "test". I think we want to remove that.

File: docs/index.md [hunk 3]
```diff
@@ -101,6 +110,32 @@ module.exports = function override(config, env) {
...
+  /* package.json */
+
+  "scripts": {
+-   "start": "react-scripts start",
++   "start": "react-app-rewired start",
+-   "build": "react-scripts build",
++   "build": "react-app-rewired build",
+    "eject": "react-scripts eject"
```
Comment (jleven) [inline docs/index.md:133]: we've lost the "test" line, it should still be here and be black and unchanged like the "eject" line.

**Comment by ramonh** [comment]
Thank you so much @jleven! I've taken care of the feedback.

**Review by jleven — APPROVED** [review]

**Pull request merged**